src/Files.App/BaseLayout.cs

```diff
@@ -248,7 +248,6 @@ internal set
 					}
 
 					NotifyPropertyChanged(nameof(SelectedItems));
-					//ItemManipulationModel.SetDragModeForItems();
```

Comment:
Why is this commented out?

Reply:
No idea, it has been introduced in #5358 but I don't know why it has been.

Reply:
Can you remove if it's not needed?

Reply:
You mean at it back the commented line?

Reply:
Remove the whole line

Reply:
Lol, I didn't see the red.

Reply:
No problem!